sample/ps256_root.pub_key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
I don't think you need root public key for anything.  I would remove it from the PR as to not confuse users.

Reply:
You're right - removed 👍